app/main.py

```diff
@@ -1,8 +1,20 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    people_list = [Person(person["name"], person["age"]) for person in people]
+    for person in people:
+        wife = person.get("wife")
```

Comment:
Line can be inserted directly to condition's. 
Also same in "husband"

Reply:
yes, but than will be bad reading code and overcharacters in line